pyteal/ast/router.py

```diff
@@ -427,15 +428,21 @@ def add_method_handler(
             raise TealInputError(
                 f"registered method {method_signature} is never executed"
             )
-        oc_create: list[EnumInt] = call_configs.oc_under_call_config(CallConfig.CREATE)
-        oc_call: list[EnumInt] = call_configs.oc_under_call_config(CallConfig.CALL)
         if method_signature in self.added_method_sig:
             raise TealInputError(f"re-registering method {method_signature} detected")
         self.added_method_sig.add(method_signature)
 
         wrapped = Router.wrap_handler(True, method_call)
 
-        if any(str(OnComplete.ClearState) == str(x) for x in oc_create):
+        def partition(cc: CallConfig) -> Tuple[bool, list[EnumInt]]:
```

Comment:
```suggestion
        def partition(cc: CallConfig) -> tuple[bool, list[EnumInt]]:
```

3.10 style type-hints